### PR TITLE
docs: added examples for awscc_signer_signing_profile

### PR DESCRIPTION
**Pull request by quixoticmonk**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

There is a deployment issue when tags are added to the resource which is called out in #1673

### CODE REVIEW

File: templates/resources/signer_signing_profile.md.tmpl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### Basic example
+
+{{ tffile (printf "examples/resources/%s/signer_signing_profile_validity.tf" .Name)}}
```
Comment (wellsiau-aws) [inline templates/resources/signer_signing_profile.md.tmpl:16]:
```suggestion
{{ tffile (printf "examples/resources/%s/signer_signing_profile.tf" .Name)}}
```

I believe this one should be the example without validity period

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
@quixoticmonk please take a look at the suggested changes, thanks!

**Comment by quixoticmonk** [comment]
> @quixoticmonk please take a look at the suggested changes, thanks!

Updated.

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thank you!

**Comment by marcosentino** [comment]
@quixoticmonk Thank you for the contribution!

**Pull request merged**